aries-site/src/components/feedback/Feedback.js

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Form,
+  Heading,
+  Layer,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const Feedback = ({
+  children,
+  modal,
+  onChange,
+  onClose,
+  onEsc,
+  onSubmit,
+  onReset,
+  show,
+  title,
+  value,
+  isSucessful,
+}) => {
+  const size = useContext(ResponsiveContext);
+
+  let content = (
+    <Box
+      fill="vertical"
+      overflow="auto"
+      width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+      pad="medium"
```

Comment:
As a Grommet component, this will be too opinionated. Needs to be props or theme driven.

Suggest adding `feedback` property to `aries.js` and use that to help set the defaults from the theme.

Reply:
I am thinking that there will be more of these type needs, but note it as something to watch out for.

Reply:
starting to think about this more and started to put examples in aries.js